Calculator/Calculator/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by MARY on 2023/05/30.
+//
+
+protocol CalculateItem { }
+
+final class CalculatorItemQueue: CalculateItem {
+    private(set) var itemQueue: [String] = []
+    
+    func enqueue(_ element: String) {
+        itemQueue.append(element)
+    }
+    
+    func dequeue() -> String? {
+        guard !itemQueue.isEmpty else { return nil }
```

Comment:
[Logical NOT Operator](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/basicoperators#Logical-NOT-Operator)를 통해 `itemQueue.isEmpty`의 값을 반전시켜줄 수 있지만 아래와 같이 표현하는 경우 가독성이 더 좋다고 느낄 수 있습니다.

```suggestion
        guard itemQueue.isEmpty == false else { return nil }
```

Reply:
네! 감사합니다 참고하겠습니다😁